### PR TITLE
Adding vpc_subnet_id to allow for t2 instances

### PR DESCRIPTION
**Pull request by hkim823**

### CODE REVIEW

File: playbooks/edx-east/edx_provision.yml [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
-    instance_type: m1.medium
-    security_group: sandbox
+    instance_type: t2.medium
+    security_group: sandbox-vpc
     # ubuntu 12.04
-    ami: ami-d0f89fb9
+    ami: ami-f478849c
     region: us-east-1
-    zone: us-east-1b
+    zone: us-east-1c
```
Comment (e0d) [inline playbooks/edx-east/edx_provision.yml:12]: What's the purpose of this change?

Comment (hkim823) [inline playbooks/edx-east/edx_provision.yml:12]: The subnet I created for sandboxes lives in 1c instead of 1b (just my mistake when I setup the VPC)

**Comment by jarv** [comment]
In general once we have the mongo server in the new vpc as well as the rds we will be connecting sandboxes directly to it so the only task needed in this play is to update server vars to point to the mongo server and rds and re-run config. There is no reason to grab backups or any of the current mongo logic.

**Comment by hkim823** [comment]
I think that makes sense, I'll edit the play to reflect what will happen in the future.

**Comment by hkim823** [comment]
@jarv this is ready for review for merging into master

**Comment by jarv** [comment]
:+1:
